Gruntfile.js

```diff
@@ -99,7 +99,8 @@ module.exports = function(grunt) {
             "phpcs",
             "phpcsfixer",
             "exec:phpstan",
-            "phpunit-runner:unit"
+            "phpunit-runner:unit",
+            "phpunit-runner:functional"
```

Comment:
@FrancisBry why is this last line here

Reply:
The functional test tests the createFromToken function and that invalid credentials throws exceptions